### DISCUSSION TITLE
Need help with MEPLHAA for Sonoff TH Elite 20A THR320D Heater Switch with Temperature Sensor

### DISCUSSION THREAD

**Discussion by mbial** [RavenSystem/esp-homekit-devices#2076] [MEPLHAA Scripts]
Any help with this device? I have the DS18B20 temperature sensor and want to use this in heater-only mode. [Here's](https://templates.blakadder.com/sonoff_THR320D.html) a Tasmota config file and [this](https://tasmota.github.io/docs/DS18x20/) is additional information about the temp sensor. I believe this sensor may require a resistor if anyone has success with it. Thanks in advance!

**Comment by PhaseAngle** [comment]
You should be able to get heating to work but the display will be a problem

**Comment by mbial** [comment]
Not concerned with the screen, really just trying to get the minimum config for heater-only using the temp sensor. GPIO settings for each of the relays and dry contact are a little less clear than the simpler, older units.

**Comment by pongista** [comment]
Please can someone help me??? I need a script for Shelly 1 like a lock mechanism but with garage door icon, Just a click to open and after 5 seconds icon state closed. Thanks to who can help me
